src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -0,0 +1,386 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+use Omnipay\Common\Message\ResponseInterface;
+
+class ApplePayAuthorizeRequest extends \Omnipay\Common\Message\AbstractRequest
+{
+    /**
+     * The class to use for the response.
+     *
+     * @var string
+     */
+    protected static $RESPONSE_CLASS = '\Omnipay\Vindicia\Message\ApplePayAuthorizeResponse';
+
+    /**
+     * Default display name for Apple Pay Session.
+     * @TODO: Add displayname to private file and add here.
+     *
+     * @var string
+     */
+    const DEFAULT_DISPLAY_NAME = 'DISPLAY_NAME_HERE';
+
+    /**
+     * @param array<string, mixed> $parameters
+     * @return ApplePayAuthorizeRequest
+     */
+    public function initialize(array $parameters = array())
+    {
+        if (!array_key_exists('displayName', $parameters)) {
+            $parameters['displayName'] = self::DEFAULT_DISPLAY_NAME;
+        }
+        parent::initialize($parameters);
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getData()
+    {
+        $this->validate('validationURL');
+
+        $data = array();
+
+        // These are from Apple's certificate – they should never change.
+        // @TODO: Add merchantIdentifier and initiativeContext to private file and add here. 
```

Comment:
there can't be any private files in omnipay repos, private info would need to be passed as a parameter